### PR TITLE
interim update of new categories

### PR DESCRIPTION
**Pull request by Bradben**
update structure to standard Azure categories.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/76c41898e8ad30c131ed7b1b76769f2f043ceef3/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/157b81c9-8bed-432f-8159-b739a0901562_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f0cdd129abdd4991a93c755cd3066626e019b566/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7994db4e-fbdc-4aeb-ada9-662dc9740913_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/TOC.yml [hunk 1]
```diff
@@ -22,21 +22,19 @@
...
+  - name: Exploring entanglement with Q#
+    href: quickstart.md
+  - name: Implement Grover's search algorithm
+    href: quickstarts/search.md
   - name: Learn Q# with the Quantum Katas
     href: intro-to-katas.md
   - name: Q# code samples
     href: https://docs.microsoft.com/en-us/samples/browse/?languages=qsharp
-- name: The QDK in-depth
+- name: Q# user's guide
```
Comment (cjgronlund) [inline articles/TOC.yml:37]:
```suggestion
- name: Q# user guide
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/19c4bab4db66e443e6ab66f836c5d52519992f36/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b64a5595-00c8-4e05-87ca-0ee4aebbe643_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by cjgronlund** [comment]
TOC top-level nodes reviewed and OK. Approved. 
This TOC will still need to incorporate content changes from Brad and Gillen - to come.

File: articles/TOC.yml [hunk 2]
```diff
@@ -46,15 +44,23 @@
...
-- name: QDK Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
+- name: Concepts 
+  items: 
+  - name: Quantum computing concepts
+    href: concepts/toc.yml
+  - name: Q# libraries
+    href: libraries/toc.yml
+- name: Reference
```
Comment (cgranade) [inline articles/TOC.yml:53]: It looks like there's now an empty Reference section as well as the "API Reference" section from the other docset; can we revert and fix before merging back to master? Thanks!

image

Comment (cgranade) [reply articles/TOC.yml:53]: +@Bradben @cjgronlund

**Pull request merged**